src/main/java/codeu/controller/BotHandler.java

```diff
@@ -0,0 +1,9 @@
+package codeu.controller;
+
+import codeu.model.data.BotRequest;
+import codeu.model.data.BotStructure;
+
+public interface BotHandler {
```

Comment:
Sometimes it's worth calling this `BotHandlerInterface` to make it clear it's not an implementation, but that's really a nit, and mostly just a suggestion to consider what is most effective in your mind